<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::MainWindow</class>
 <widget class="QMainWindow" name="Otter::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Otter</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="actionNewPrivateTab"/>
    <addaction name="actionNewWindow"/>
    <addaction name="actionNewPrivateWindow"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionCloseTab"/>
    <addaction name="separator"/>
    <addaction name="actionSessions"/>
    <addaction name="actionImportAndExport"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="separator"/>
    <addaction name="actionWorkOffline"/>
    <addaction name="separator"/>
    <addaction name="actionShowMenubar"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionAddBookmark"/>
    <addaction name="actionManageBookmarks"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFeeds">
    <property name="title">
     <string>Feeds</string>
    </property>
    <addaction name="actionReadFeeds"/>
    <addaction name="actionManageFeeds"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuFeeds"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="panelWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="panelWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="tabsWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="tabsWidgetContents"/>
  </widget>
  <action name="actionNewTab">
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="actionNewPrivateTab">
   <property name="text">
    <string>New Private Tab</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionNewPrivateWindow">
   <property name="text">
    <string>New Private Window</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSessions">
   <property name="text">
    <string>Sessions</string>
   </property>
  </action>
  <action name="actionImportAndExport">
   <property name="text">
    <string>Import and Export</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionWorkOffline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Work Offline</string>
   </property>
  </action>
  <action name="actionShowMenubar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Menubar</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>Close Tab</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="text">
    <string>Add Bookmark....</string>
   </property>
  </action>
  <action name="actionManageBookmarks">
   <property name="text">
    <string>Manage Bookmarks...</string>
   </property>
  </action>
  <action name="actionReadFeeds">
   <property name="text">
    <string>Read Feeds</string>
   </property>
  </action>
  <action name="actionManageFeeds">
   <property name="text">
    <string>Manage Feeds...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
